packages/transaction-log/src/interfaces/index.interface.ts

```diff
@@ -65,10 +65,26 @@ export interface CustomizeDialogProps {
   open: boolean
   onClose: () => void
   onApply: () => void
-  visibleFilters: {[key: string]: boolean}
+  visibleFilters: {
+    status: boolean
+    statusCode: boolean
+    channel: boolean
+    startDate: boolean
+    endDate: boolean
+    limit: boolean
+    reruns: boolean
+    host: boolean
+    port: boolean
+    path: boolean
+    param: boolean
+    client: boolean
+    method: boolean
+  }
   handleFilterVisibilityChange: (
     event: React.ChangeEvent<HTMLInputElement>
   ) => void
+  onRestoreDefaults: () => void
+  isDefaultState: boolean
```

Comment:
```suggestion
  isDefaultState?: boolean
```

Reply:
Why would you want to make the property optional? @marvinIsSacul